RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
```

Comment:
Does this change fits the `patch` type?  Feels like at least `minor` one as it affects the legacy django users. However, I'm not sure about this since nothing is broken. If it's ok than it's ok.

Reply:
FWIW, the last release to change Django support was also a patch release (3.44.7), so there’s a precedent here.

Reply:
Ah, fine then.

Reply:
Well, we removed support for Django 1.10 in Hypothesis 3.44.7, so there's precedent.  IMO patch is the correct level, as we're just updating our build metadata to reflect documented support policies.

Reply:
I think maybe we should do a minor release (and specify this in our guides). Although there's precedent of us doing this, we should probably be compatible with the rules about not introducing deprecations in patch releases.

Reply:
Happy to go to a minor release.

FYI though there are no new deprecations in this - in fact no code changes at all.  We just declare a different compatibility range and stop testing the old version.

Reply:
Yes I realise that, but my point is that stopping working with a version that previously worked is in some sense at least as big a change as a deprecation.